modules/ROOT/pages/clustering/setup/single-to-cluster.adoc

```diff
@@ -0,0 +1,18 @@
+:description: This section describes how to move from a single `system` primary deployment to a cluster.
+[role=enterprise-edition]
+[[single-to-cluster]]
+= Move from a single `system` primary to a cluster deployment
+
+This is an example of how to move from a deployment with a single `system` primary to a cluster with multiple `system` primaries.
+
+== Guide
```

Comment:
This doesn't follow the structure elsewhere in the clustering chapter. I will take a stab at rewriting it tomorrow.